pyqt-apps/siriushla/as_ps_commands/conn.py

```diff
@@ -0,0 +1,681 @@
+"""Conn module."""
```

Comment:
I think this module as a whole, which has nothing to do with GUI, can be available for use by other layers without Qt/PyDM and should be implemented somewhere within `siriuspy` package.

Reply:
done, @xresende ! Please take a look: https://github.com/lnls-sirius/dev-packages/pull/899